test/functional/neutronless/vcmp/test_vcmp.py

```diff
@@ -350,7 +350,7 @@ def test_vcmp_delete_listener(
             'ClusterManager')
 @mock.patch('f5_openstack_agent.lbaasv2.drivers.bigip.vcmp.LOG')
```

Comment:
Do these tests run?  This is fine, but I am not sure what the deal with these tests is.

Reply:
@richbrowne Not certain... however, they will pass with my other PR that I'll be creating shortly that will handle the ssh cmd properly in the tracker.